src/api/lib/tasks/statistics/current.rake

```diff
@@ -9,12 +9,12 @@ namespace :statistics do
     time = elements.sum { |element| element[1].to_i - element[0].to_i }
     time_in_hours = time / elements.count / 60 / 60
 
-    information = ''"OBS STATISTICS ON #{Time.zone.today}\n
```

Comment:
This one is weird...

Reply:
I checked the commit history, used to be just `""" """` then it got corrected to `''" ""'` because of a rubocop was coplaining about not using single quotes :D :D